### ISSUE TITLE
Distribute latest version

### ISSUE THREAD

**Issue by ad-si** [archaeron/purescript-yaml#8]
- [x] Rename repo to `purescript-yaml-next` to match name on bower
- [x] Publish new version to Pursuit as described in https://pursuit.purescript.org/help/authors
- [x] Submit it to https://github.com/purescript/package-sets

Let me know if you need help with anything 😊

**Comment by dgendill** [comment]
Hi, Adrian, I'm not sure how to approach some of these tasks. Maybe you could help identify first steps?

First, I don't think I have access to rename this repo. Maybe @archaeron could transfer ownership to you, me, or the  [purescripters group](https://github.com/purescripters) (or whatever group is active in the community)?

After the repo has been renamed, I'm guessing we'd have a repo named either...

- https://github.com/purescripters/purescript-yaml-next
- https://github.com/archaeron/purescript-yaml-next
- https://github.com/dgendill/purescript-yaml-next
- https://github.com/ad-si/purescript-yaml-next

Once the renaming has been taken care of would we create a PR into https://github.com/purescript/package-sets/package.json? Something like this...

```
{
    "yaml-next": {
        "dependencies": [
            "argonaut-codecs",
            "argonaut-core",
            "console",
            "effect",
            "foreign",
            "foreign-generic",
            "functions",
            "ordered-collections",
            "psci-support",
            "spec",
            "unsafe-coerce"
        ],
        "repo": "RENAMED REPO URL.git",
        "version": "v2.0.0"
}
```

After yaml-next has been added to the package set, would we add a psc-package.json file into the purescript-yaml-next repo? Something like...

```
{
    "name": "yaml-next",
    "set": "psc-0.13.3-20191005",
    "source": "https://github.com/purescript/package-sets.git",
    "depends": [
        "argonaut-codecs",
        "argonaut-core",
        "console",
        "effect",
        "foreign",
        "foreign-generic",
        "functions",
        "ordered-collections",
        "psci-support",
        "spec",
        "unsafe-coerce"
    ]
}
```
  
After adding the psc-package.json file I was able to run `psc-package install`. And the reason for running that command was so I could generate the HTML docs. Without doing that I got this error using pulp...

```
$ pulp docs -- --format html
* Generating documentation in /purescript-yaml
.psc-package/psc-0.13.3-20191005/.set/packages.json does not exist
* ERROR: Subcommand terminated with exit code 1
```

And without adding psc-package, I got this error...

```
$ pulp docs -- --format html
* ERROR: No bower.json or psc-package.json found in current or parent directories. Are you in a PureScript project?
```

My other question is, do we still need the bower.json file? Looking at the [CONTRIBUTING.md](https://github.com/purescript/package-sets/blob/master/CONTRIBUTING.md) file in the package-sets repo it says "All packages that are included here must first be published via bower with no exceptions." Is that still the case?

Thanks!

**Comment by ad-si** [comment]
> Maybe @archaeron could transfer ownership to you, me, or the purescripters group (or whatever group is active in the community)?

Good idea! I guess  https://github.com/purescript-contrib would be best

> psc-package.json file into the purescript-yaml-next repo

No, I don't think that 's necessary. Spago handles all of that. HTML docs can be generated with `spago docs`. 

> do we still need the bower.json file

Yes, still necessary, but `spago bump-version` does all the necessary steps for distribution automatically, so it's not a big issue…

**Comment by ad-si** [comment]
@garyb Would purescript-contrib be interested in hosting this repository? (Of course only if that's Ok for @archaeron !?)

**Comment by archaeron** [comment]
Hi
Sorry I‘m on holidays and didn’t see your message earlier :)
Sure I‘d be happy to transfer the repo

**Comment by ad-si** [comment]
@archaeron The @purescript-contrib community doesn't want to take it over for the time being. Maybe you can just add me as contributor and we can clean it up right here?

**Comment by archaeron** [comment]
@ad-si I have added you as collaborator

**Comment by ad-si** [comment]
Awesome, thanks! Can you still help me with the biggest change: Rename it to `purescript-yaml-next`. I can do the rest

**Comment by archaeron** [comment]
will this break old scripts?

**Comment by ad-si** [comment]
As far as I can tell not. Bower already uses the -next name and GitHub also creates a redirection from the old repo URL. I renamed the GitHub repo for a bower package once and it still works…

**Comment by archaeron** [comment]
done

**Comment by ad-si** [comment]
I just released a new version (3.1.0) and it worked without problems. So we can close this issue.
